components/infobox/commons/infobox_team.lua

```diff
@@ -232,7 +258,7 @@ function Team:_setLpdbData(args, links)
 		disbanddate = args.disbanded,
 		coach = args.coaches,
 		manager = args.manager,
-		region = args.region,
+		region = self:getStandardRegionValue(args.region, args.location),
```

Comment:
due to this the sc2 /Custom needs adjusting too (setting `lpdb.region = nil`)

Reply:
![bild](https://user-images.githubusercontent.com/3426850/147461679-4fa23a4c-1367-407e-ac82-74ab92613742.png)

SC2 custom also does this?

Reply:
oh right, then nvm^^